git-credential-msal

```diff
@@ -182,8 +181,9 @@ def msal_acquire_oidc_id_token(
 ) -> str:
     scopes = ["email openid User.Read"]
     id_token = None
-    cache = get_msal_cache()
-    http_cache = get_http_cache()
+    keyring_name = f"git-credential-msal_{tenant_id}_{client_id}"
```

Comment:
Thinking about this some more, would it make sense to create a `git-credential-msal` directory and put `{tenant_id}_{client_id}` inside there? Otherwise we'll litter the .cache directory with a bunch of files.

Reply:
Good point. Will make that change.